ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -164,24 +166,30 @@ const Footer = ({ setStepNotification }) => {
                     );
 
                     return (
-                        <>
-                            <Button
-                              variant="primary"
-                              isDisabled={isInProgress}
-                              isLoading={isInProgress}
-                              onClick={() => goToStep(activeStep, onNext)}>
-                                {nextButtonText}
-                            </Button>
-                            <Button
-                              variant="secondary"
-                              isDisabled={isBackDisabled}
-                              onClick={onBack}>
-                                {_("Back")}
-                            </Button>
-                            <Button variant="link">
-                                {_("Quit")}
-                            </Button>
-                        </>
+                        <Stack hasGutter>
+                            {activeStep.id === "review-configuration" &&
+                            <ReviewConfigurationFooter
+                              setIsEraseChecked={value => setIsNextDisabled(!value)}
+                              isEraseChecked={!isNextDisabled} />}
+                            <ActionList>
+                                <Button
+                                  variant="primary"
+                                  isDisabled={isInProgress || (["review-configuration"].includes(activeStep.id) && isNextDisabled)}
```

Comment:
Why is the check for includes like this? Wouldn't it be better to check if ``activeStep.id == "review-configuration``?

Reply:
Sure, but if more 'steps' in the future will need this kinda of guarding the 'Next' with a checkbox or dialog, this will need to be reverted to what @M4rtinK uses now.

Reply:
ok, it give sense.